python/pyspark/pandas/frame.py

```diff
@@ -4871,7 +4868,14 @@ def round(self, decimals: Union[int, Dict[Name, int], "Series"] = 0) -> "DataFra
         else:
             raise TypeError("decimals must be an integer, a dict-like or a Series")
 
-        def op(psser: ps.Series) -> Union[ps.Series, GenericColumn]:
+        if is_remote():
+            from pyspark.sql.connect.column import Column as ConnectColumn
+
+            Column = ConnectColumn
+        else:
+            Column = PySparkColumn  # type: ignore[assignment]
```

Comment:
Can we remove this, and just use `PySparkColumn`?

Reply:
It complains `UnboundLocalError: local variable 'Column' referenced before assignment` if we don't explicitly assign the `PySparkColumn` to the variable here, so I fixed it.

Reply:
Can you use `PySparkColumn` for type hint instead

Reply:
Oh, I missed that it's used only for type hint

Reply:
Just updated, thanks!